src/stepper/Stepper.tsx

```diff
@@ -196,6 +199,7 @@ export default class Stepper extends PureComponent<StepperProps, StepperState> {
           status={ButtonStatus.UNSTYLED}
           disabled={isMin}
           isBubble
+          focus={!isMin && isMax && focusFirstInteractiveElement}
```

Comment:
If `props.value` is strictly between `min` and `max` it looks like none of the buttons will be focused, is this expected?

Reply:
You're right, was working on my case because by default the minus button was disabled. I'll try to find a better solution